pkg/sfu/buffer/buffer.go

```diff
@@ -264,6 +264,14 @@ func (b *Buffer) Bind(params webrtc.RTPParameters, codec webrtc.RTPCodecCapabili
 				}
 			}
 		}
+		if bitrates > 0 {
+			pps := bitrates / 8 / 1200
+			for pps > b.bucket.Capacity() {
+				if b.bucket.Grow() >= b.maxVideoPkts {
```

Comment:
Is the max set to accommodate high bit rates?

Reply:
The max is set in the config to avoid incorrect publisher settings holding a lot of server memory.